models/spring-ai-ollama/src/main/java/org/springframework/ai/ollama/OllamaChatModel.java

```diff
@@ -198,7 +213,8 @@ public Flux<ChatResponse> stream(Prompt prompt) {
 
 			observation.parentObservation(contextView.getOrDefault(ObservationThreadLocalAccessor.KEY, null)).start();
 
-			Flux<OllamaApi.ChatResponse> ollamaResponse = this.chatApi.streamingChat(request);
+			Flux<OllamaApi.ChatResponse> ollamaResponse = this.retryTemplate
```

Comment:
When in streaming mode, we don't use the `RetryTemplate` as it's not gonna work with the reactive streams. There is pending design on how to configure retries for streaming. You can follow this issue for updates: https://github.com/spring-projects/spring-ai/issues/1193